.dockerignore

```diff
@@ -0,0 +1 @@
+.github
```

Comment:
Why is this needed?

Reply:
This is a small optimization for local testing. I can exclude it from the PR if necessary. On line 63 in Dockerfile.runtest, we have COPY ./ ./, which copies all files. This just excludes it from the Docker context and doesn’t copy this folder to the image at this step. This means that if I change something in the .github folder, it won’t invalidate all Docker layers after COPY ./ ./.

Reply:
Gotcha, I was just wondering if ignoring `.github`, where workflows are declared, has anything to do with workflows no longer running.